docs/pages/store/tables.mdx

```diff
@@ -46,6 +46,10 @@ The table ID encodes the table's type and name in a 32-byte `ResourceId` value a
 import { ResourceId, ResourceIdLib } from "@latticexyz/store/src/ResourceId.sol";
 import { RESOURCE_TABLE, RESOURCE_OFFCHAIN_TABLE } from "@latticexyz/store/src/storeResourceTypes.sol";
 
+.
+.
+.
+
```

Comment:
I don't think we need this?

Reply:
It signifies a change of context, from the header matter to inside a function inside the contract. I changed it to horizontal dots so it'll take less space.

Reply:
It signifies a change of context from the header stuff to inside a function inside the contract. I changed it to a horizontal ... so it'll take less space.